controllers/controller_shared_test.go

```diff
@@ -26,9 +26,26 @@ import (
 	"github.com/stretchr/testify/assert"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var instanceSelectorNoMatchingInstances = v1beta1.GrafanaCommonSpec{
+	InstanceSelector: &metav1.LabelSelector{
+		MatchLabels: map[string]string{"test": "no-matching-instances"},
+	},
+}
+
+func requestFromMeta(obj metav1.ObjectMeta) ctrl.Request {
+	return ctrl.Request{
```

Comment:
Such functions should be marked as helpers (call to `t.Helper()` after `t *testing.T` is passed to the function).

Reply:
Yeah, I think it should be fine :)